ludwig/features/numerical_feature.py

```diff
@@ -267,7 +299,37 @@ def postprocess_predictions(
             skip_save_unprocessed_output = True
 
         if PREDICTIONS in predictions and len(predictions[PREDICTIONS]) > 0:
-            postprocessed[PREDICTIONS] = predictions[PREDICTIONS].numpy()
+            if metadata['preprocessing']['normalization'] is not None:
+                normalization_type = metadata['preprocessing']['normalization']
+                NumericTransformer = get_from_registry(
+                    normalization_type,
+                    numeric_transformation_registry
+                )
+                if normalization_type == 'zscore':
+                    numeric_transformer = NumericTransformer(
+                        metadata['mean'],
+                        metadata['std']
+                    )
+                elif normalization_type == 'minmax':
+                    numeric_transformer = NumericTransformer(
+                        metadata['min'],
+                        metadata['max']
+                    )
+                elif normalization_type == 'log1p':
+                    numeric_transformer = NumericTransformer()
+                else:
+                    raise ValueError(
+                        'Normalization "{}" not supported. Valid values are '
+                        '"minmax" or "zscore"'.format(normalization_type)
```

Comment:
Same thing as before regarding how to remove this if.

Reply:
done